src/overlays/actors/ovl_Bg_Gnd_Firemeiro/z_bg_gnd_firemeiro.c

```diff
@@ -19,8 +20,8 @@ void func_808795AC(BgGndFiremeiro* this, GlobalContext* globalCtx);
 void func_80879668(BgGndFiremeiro* this, GlobalContext* globalCtx);
 void func_80879808(BgGndFiremeiro* this, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_0600E210;
-extern UNK_TYPE D_0600ECD8;
+extern Gfx gFireTrialPlatformDL[];
+extern CollisionHeader gFireTrialPlatformCol;
```

Comment:
Why is the object header not used here?

Reply:
It gave some error when I tried using the header, though unfortunately I don't remember what.

Reply:
not sure what the issue was but its fine now